src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -107,4 +156,31 @@ describe('<TranscriptsListItemInfo /', () => {
     userEvent.click(proteinLink);
     expect(defaultProps.onProteinLinkClick).toHaveBeenCalled();
   });
+
+  it('display metadata when it is available', () => {
+    const { queryByText } = renderComponent({
+      transcript: createGencodeBasicTranscript(),
+      expandMoreInfo: true
+    });
+    const refseqText = queryByText('gencode basic');
+    expect(refseqText).toBeTruthy();
+  });
+
+  it('display CCDS when it is available in external references', () => {
+    const { queryByText } = renderComponent({
+      transcript: createCCDSXrefTranscript(),
+      expandMoreInfo: true
+    });
+    const refseqText = queryByText('CCDS');
+    expect(refseqText).toBeTruthy();
+  });
+
+  it('display Refseq when ncbi_transcript is available', () => {
+    const { queryByText } = renderComponent({
+      transcript: createMANETranscript(),
+      expandMoreInfo: true
+    });
+    const refseqText = queryByText('RefSeq');
+    expect(refseqText).toBeTruthy();
```

Comment:
you may want to check that the component is displaying a link to the NCBI transcript here